packages/contracts/contracts/TestContracts/invariants/echidna/EchidnaDoomsdayTester.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.17;
+
+import "./EchidnaAsserts.sol";
+import "./EchidnaProperties.sol";
+import "../TargetFunctions.sol";
+
+contract EchidnaDoomsdayTester is EchidnaAsserts, EchidnaProperties, TargetFunctions {
+    constructor() payable {
+        _setUp();
+        _setUpActors();
+        _setupVictim();
+    }
+
+    address VICTIM = address(0x5000000000000005);
+    address DAO = address(0xb453d);
+    bytes32[] victimCdps;
+
+    // NOTE: Customized setup for victim
+    function _setupVictim() internal {
+        bool success;
+        address[] memory tokens = new address[](2);
+        tokens[0] = address(eBTCToken);
+        tokens[1] = address(collateral);
+        address[] memory callers = new address[](2);
+        callers[0] = address(borrowerOperations);
+        callers[1] = address(activePool);
+        address[] memory addresses = new address[](2);
+        addresses[0] = VICTIM;
+        addresses[1] = DAO;
+        Actor[] memory actorsArray = new Actor[](2);
+        for (uint i = 0; i < 2; i++) {
+            actors[addresses[i]] = new Actor(tokens, callers);
+            (success, ) = address(actors[addresses[i]]).call{value: INITIAL_ETH_BALANCE}("");
+            assert(success);
+            (success, ) = actors[addresses[i]].proxy(
+                address(collateral),
+                abi.encodeWithSelector(CollateralTokenTester.deposit.selector, ""),
+                INITIAL_COLL_BALANCE
+            );
+            assert(success);
+        }
+
+        priceFeedMock.setPrice(1e8); /// TODO: Does this price make any sense?
```

Comment:
as a reviewer not sure as I'm not sure what the price is trying to achieve.

Reply:
Please do review the amount opened on the CDP

I was having issues in minting 1e18+ eBTC